### PR TITLE
NIFI-6524 MergeContent properties should accept expression language variables

### PR DESCRIPTION
**Pull request by adarmiento**
### Description of PR

The following `MergeContent` properties support expression language:
- Minimum Number of Entries: Accepts from Variable Registry
- Maximum Number of Entries: Accepts from Variable Registry
- Minimum Group Size: Accepts from Variable Registry
- Maximum Group Size: Accepts from Variable Registry
- Max Bin Age: Accepts from Variable Registry
- Maximum number of Bins: Accepts from Variable Registry

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with **NIFI-XXXX** where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?

- [x] Is your initial contribution a single, squashed commit? _Additional commits in response to PR reviewer feedback should be made on this branch and pushed to allow change tracking. Do not `squash` or use `--force` when pushing to allow for clean monitoring of changes._

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via `mvn -Pcontrib-check clean install` at the root `nifi` folder?
- [x] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the `LICENSE` file, including the main `LICENSE` file under `nifi-assembly`?
- [ ] If applicable, have you updated the `NOTICE` file, including the main `NOTICE` file found under `nifi-assembly`?
- [ ] If adding new Properties, have you added `.displayName` in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/MergeContent.java [hunk 1]
```diff
@@ -393,6 +393,7 @@
                 results.add(StandardValidators.FILE_EXISTS_VALIDATOR.validate(DEMARCATOR.getName(), demarcatorValue, context));
             }
         }
+
```
Comment (AxelSync) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/MergeContent.java]:
```suggestion
```

**Comment by AxelSync** [comment]
cc @mgaido91 
cc @ijokarumawak 
Could you please review this PR? Thanks

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestMergeContent.java [hunk 2]
```diff
@@ -1153,6 +1153,248 @@ public void testMaxBinAge() throws InterruptedException {
         runner.run();
 
         runner.assertTransferCount(MergeContent.REL_MERGED, 1);
+
+        runner = TestRunners.newTestRunner(new MergeContent());
```
Comment (mgaido91) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestMergeContent.java]: why not having this in a dedicated test case for this PR which tests all the attributes EL?

Comment (adarmiento) [reply nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestMergeContent.java]: Should I only check the EL values are correctly evaluated, or also that the processor works when using the EL attributes? I found the tests redundant too, but I wasn't sure about checking only that the `evaluateExpression()` worked as expected was enough

**Pull request closed without merging**